src/context/Baser.spec.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import assert from 'assert'
```

Comment:
>port assert [](start = 2, length = 11)

Are .spec files tests? Is it possible to put them in a different directory?

Reply:
`spec` files are tests. This is Mocha's convention, not mine.

By default I keep unit tests (and other related files such as CSS) next to the file it is testing. High-level tests would go in a different directory. I've seen both conventions at Microsoft. For example here at [Office Fabric aka FluentUI](https://github.com/microsoft/fluentui/tree/master/packages/experiments/src/components/Button). Let's discuss.

Reply:
In transparency: I do like having the unit tests next to the associated source files. 1) It helps me keep them in sync. 2) this pattern is already being used (as mentioned) with CSS.